CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
 # Change Log
+
+## 0.4.2
+### Changed
+- Extra implementations of the primary task will now be hidden in tools that support implementations and the 'private' property (like Bolt).
+
```

Comment:
- Only use facter to compute facts when the `puppet-agent` feature is available on target in the ruby implementation.